frontend/.env

```diff
@@ -0,0 +1,2 @@
+REACT_APP_SUPABASE_URL = https://yyuyncpblcnwlrfpvenq.functions.supabase.co/sign-up
```

Comment:
Indeed, we missed this! Those credentials were from a testing env, not the official one we'll be using. I've added .env to gitignore too, just in case.